drf/university/models.py

```diff
@@ -1,20 +1,21 @@
-from localflavor.us.models import USStateField
-from django_countries.fields import CountryField
 from django.db import models
-from pkg.models import CommonDates
+from pkg.models import AbstractModel
+from school.models import School
 
 
 def upload_logo_image(instance, filename):
     return "university/{unv}/{filename}".format(unv=instance.name,
                                                 filename=filename)
 
 
-class University(CommonDates):
-    name = models.CharField(max_length=50)
-    address = models.CharField(max_length=50, null=True, blank=True)
-    city = models.CharField(max_length=30, null=True, blank=True)
-    country = CountryField(null=True, blank=True)
-    state = USStateField(null=True, blank=True)
-    description = models.TextField(max_length=200, null=True, blank=True)
+class University(AbstractModel):
+    schools = models.ManyToManyField(School, related_name='universities')
```

Comment:
I think you miss through model here. Because that's the place where you will put _is_main_school_ field